cabwiz/cutlist.py

```diff
@@ -368,7 +368,99 @@ def isometric_view(job):
     result.add(arr)
     return result
 
+def elevation_view(job):
 
+    elevationLines = [
```

Comment:
Lee, the new lines added to elevationLines and below all should not be in this commit. They are not related to the depth dimension issue. I suppose they got into the commit accidentally. So, we need to figure out how to move them to a different commit, in a new topic branch specifically for the topic they relate to. At the moment, I don't know how to do that. Will let you know if I figure out how. If you do, will you let me know? Thanks.